squire_lib/src/accounts.rs

```diff
@@ -0,0 +1,20 @@
+use uuid::Uuid;
+use settings;
+
+struct squire_account {
+    display_name: String,
+    user_name: String,
+    user_id: Uuid,
+    do_share: bool,
+}
+
+struct organization_account {
+    display_name: String,
+    user_name: String,
+    user_id: Uuid,
+    do_share: bool,
```

Comment:
I don't think we need this field. Organizations will not have the ability to withhold results for their tournaments. We only give players the ability to anonymize their data. We won't be affording TOs the same option.